src/style.css

```diff
@@ -1,3 +1,106 @@
+html {
+  margin: 0;
+  font-family: 'Roboto ', sans-serif;
+  font-size: 12px;
+
+  --text-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  position: relative;
+  align-items: center;
+  width: 1024px;
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it to avoid overflow or accidental scroll bar.